src/renderer/src/assets/main.css

```
@@ -247,6 +247,21 @@
   --tab-group-split-divider-strong: #71717a;
 }
 
+/* Why: app ships no CJK font; Chinese used to hit mixed OS sans-serif fallbacks
+   (uneven stroke weight). Pin one CJK family per Chinese locale. */
+:root:lang(zh) {
+  --app-font-family:
+    'Geist', -apple-system, BlinkMacSystemFont, 'Segoe UI', 'PingFang SC', 'Microsoft YaHei',
```

Stylelint (17.14.0)

[error] Expected "BlinkMacSystemFont" to be "blinkmacsystemfont" (value-keyword-case)

(value-keyword-case)